addons/stock/models/stock_picking.py

```diff
@@ -1192,8 +1192,8 @@ def _get_without_quantities_error_message(self):
         :rtype: str
         """
         return _(
-            'You cannot validate a transfer if no quantities are reserved. '
-            'To force the transfer, encode quantities.'
+            "Transfer trouble alert! 0 quantity transfer? That's like trying to move air around!\n"
+            "Set the quantities, and let the magic happen!"
```

Comment:
I think this sounds better. Also, I'm not a big fan of the "trying to move air around" joke since some business may literally be selling/moving (tanks of) air, but I can't think of a better one to replace it with.

```suggestion
            "Trouble alert! Transferring no quantities? That's like trying to move air around!\n"
            "Set some quantities, and let the magic happen!"
```